src/redis-benchmark.c

```diff
@@ -1408,6 +1502,13 @@ int parseOptions(int argc, char **argv) {
         if (!strcmp(argv[i],"-c")) {
             if (lastarg) goto invalid;
             config.numclients = atoi(argv[++i]);
+        } else if (!strcmp(argv[i], "--dist")) {
+            if (lastarg) goto invalid;
+            config.distribution = atoi(argv[++i]);
+        } else if (!strcmp(argv[i], "--qps")) {
```

Comment:
I think it is more appropriate to use `--throughput` because `redis-benchmark` uses this word

Reply:
I think there is a little bit difference between qps and throughput. Qps here refers to the requests load created by benchmark (what we want to control), throughput refers to the actual processed requests per second. So I think qps is more appropriate here.